fc/parsing/CompactSyntaxParser.py

```diff
@@ -1115,39 +746,15 @@ def GetValue(self, token, negate=False):
                         result = '-' + result
             return result
 
-        def _xml(self):
```

Comment:
Again, document what's useful. We'll need to generate pint unit definitions from these, probably.